_notices/rsn0037.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 37 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "Deprecation of `CentOS 7` in v24.06"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v24.04"
+notice_created: 2024-02-14
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2024-02-14
+---
+
+## Overview
+
+RAPIDS is deprecating `CentOS 7` in release v24.04, and it will be removed in release v24.06.  This is due to `CentOS7` becoming end of life in June 2024, which means RAPIDS will no longer publish any images built on `CentOS 7` following the RAPIDS `v24.04` release.
+
+
+## Impact
+
+Users should consider switching to any of the following supported operating systems for any rapidsai/rapidsai image:
```

Comment:
```suggestion
Users should consider installation via conda or pip, or switching to any of the following supported operating systems for any rapidsai/rapidsai image:
```

Reply:
@aravenel I would not recommend making this change. We may not be able to support CentOS 7 via conda or pip packages forever -- we may have to adopt a higher glibc version as the basis for RAPIDS if build infrastructure (like conda-forge) or core dependencies (like Arrow) drop glibc 2.17 support. At that point, conda/pip packages would not support CentOS 7.